lib/timeline.js

```diff
@@ -164,6 +192,38 @@ Timeline.prototype.unsubscribe = function (userToken, topic, callback) {
   this._request(opts, callback);
 };
 
+/**
+ * List user subscriptions
+ *
+ * @param {String} userToken
+ * @callback {callback} callback
+ */
+Timeline.prototype.listSubscriptions = function (userToken, callback) {
+  if (typeof userToken !== 'string') {
+    return process.nextTick(function () {
+      callback(new Error('Expected userToken to be a string.'));
+    });
+  }
+
+  var opts = {
+    method: 'GET',
+    endpoint: '/v1/user/subscriptions',
+    headers: {
+      'X-User-Token': userToken
+    }
+  };
+
+  this._request(opts, function (err, body, res) {
+    if (err) {
+      return callback(err);
+    }
+
+    var topics = body.topics || [];
+
+    callback(null, topics, res);
```

Comment:
to keep the api consistent with the request module, should we set `body.topics` to an array if it doesn't exist and respond with `body` instead of passing the array directly?

```js
if (!Array.isArray(body.topics)) body.topics = [];
callback(null, body, res);
```